content/documents/_index.md

```diff
@@ -25,7 +22,7 @@ A large amount of detailed documentation is available for users and developers.
 * [Human Interface Guidelines](/docs/HIG/index.xml)
 * [Icon Guidelines](/development/icon-guidelines)
 
-### Legacy Documentation
+### BeOS-Era Documentation
```

Comment:
"Legacy" is clearer to me. No one knows when the BeOS era was, or will be.

Reply:
Nobody knows what legacy BeOS documents are either, but I edited that because a person could possibly connect "Legacy" and "BeOS" together.